ofrak_patch_maker/ofrak_patch_maker_test/test_toolchain_c.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-from ofrak.core.architecture import ProgramAttributes
+from ofrak.core.architecture import ProgramAttributes, InstructionSet
```

Comment:
`InstructionSet` is now in `ofrak_type`.

Reply:
Fixed